views/proposals.tmpl

```diff
@@ -138,29 +148,45 @@
                                         In Discussion
                                     </span>
                                 {{else if eq (len .VoteResults) 0 }}
-                                    <span class="text-progress">
+                                    <span class="text-progress position-relative" data-tooltip="Voting in progress">
                                         In Progress
                                     </span>
                                 {{else}}
                                     {{range $i, $vr := .VoteResults}}
                                         {{if eq $vr.Option.OptionID "yes"}}
                                             {{$votesPercent :=  percentage $vr.VotesReceived $v.TotalVotes}}
-                                            {{if lt $.Tip.Height (toInt $v.Endheight)}}
+                                            {{$blocksRemaining := subtract (int64 (toInt $v.Endheight)) (int64 $.Tip.Height)}}
```

Comment:
I know it has come up before, but `Endheight` should really already be an integer type. Probably outside the scope of this PR.

Reply:
This conversion should be a PR on its own as data from back-end returns the field as a string. It would be optimal to change `EndHeight` string value to integer during the API data unmarshaling.